relm4-macros/src/token_streams.rs

```diff
@@ -144,7 +144,9 @@ impl Widget {
         if generate_init_root_stream {
             if let Some(root_name) = root_name {
                 streams.rename_root.extend(quote_spanned! {
-                    name_span => let #name = #root_name.clone();
```

Comment:
Could the issue be caused by the fact that the expanded tokens use the span of the name here? Clippy shouldn't warn if the span of the warning is part of the macro.

Reply:
True, good catch. It can be tempting to use use attributes, but sometimes the fix is even easier. Most importantly, we now have a test case though that ensures we keep this edge case covered.